charts/opensearch/values.yaml

```diff
@@ -323,12 +323,22 @@ terminationGracePeriod: 120
 
 sysctlVmMaxMapCount: 262144
 
+clusterHealthCheckParams: "wait_for_status=green&timeout=1s"
+
+startupProbe:
+  enabled: false
+  tcpSocket:
+    port: 9200
+  initialDelaySeconds: 5
+  periodSeconds: 10
+  timeoutSeconds: 3
+  failureThreshold: 30
 readinessProbe:
+  tcpSocket:
+    port: 9200
+  periodSeconds: 5
+  timeoutSeconds: 3
   failureThreshold: 3
```

Comment:
This is the default value so can be removed.

Reply:
Just added it for good experience. Good to mention this explicitly so that the novice users know what is happening. Opinions?

Reply:
I am ok with it staying there, even thought it is defaults seems like good references.